Include `CASE` expression breaking change in `v1.5` (#1683)
risingwavelabs/risingwave-docs: src/pages/release-notes.md

  - Supports `to_jsonb`. [#13161](https://github.com/risingwavelabs/risingwave/pull/13161). See [JSON functions and operators](/docs/current/sql-function-json).
  - Supports JSON path operators and functions. [#13568](https://github.com/risingwavelabs/risingwave/pull/13568). See [JSON functions and operators](/docs/current/sql-function-json).
  - Supports array operators `@>` and `<@`. [#13253](https://github.com/risingwavelabs/risingwave/pull/13253). See [Array functions and operators](/docs/current/sql-function-array).

>   - Fixes correctness of case expression. Previously if there were multiple matching values, the last one would match. [#13890](https://github.com/risingwavelabs/risingwave/pull/13890). **This is a breaking change**. Any materialized views which contain `CASE` expressions should be dropped and recreated.

Reviewer: ```suggestion
  - Fixes the correctness of case expressions. Previously if there were multiple matching values, the last one would match. [#13890](https://github.com/risingwavelabs/risingwave/pull/13890). **The fix introduces a breaking change**. It is recommended to drop and recreate any materialized views that contain `CASE` expressions.
```

>     If your system enters a crash-loop, we suggest upgrading to v1.5.2, and dropping the corresponding materialized view with `CASE`.

Reviewer: ```suggestion
    If your system enters a crash-loop, we suggest upgrading to v1.5.2, and dropping the corresponding materialized view that contains `CASE` expressions.
```
